src/main/webapp/app/shared/service/sort.service.ts

```diff
@@ -12,7 +12,7 @@ export class SortService {
             const valueA = this.customGet(a, key, undefined);
             const valueB = this.customGet(b, key, undefined);
 
-            if (!valueA || !valueB) {
+            if ((!valueA && valueA !== 0) || (!valueB && valueB !== 0)) {
```

Comment:
I fixed this differently in another PR

Reply:
ah yea, the `== undefined` is definitely cleaner